### ISSUE TITLE
Support d.ts files

### ISSUE THREAD

**Issue by ry** [dsherret/dprint#100]
https://github.com/denoland/deno/pull/3988#discussion_r379083551

cc @bartlomieju

**Comment by dsherret** [comment]
Heh, this was an oversight on my part. Should be an easy fix.

**Comment by dsherret** [comment]
@bartlomieju was there a specific scenario you found that wasn't working? I think dprint should work with .d.ts files.

That said, I found two small issues when I was looking into this and they are published in 0.5.1.

---

Forgot to add tests for this one:

```ts
global {
}
// incorrectly formatted as...
global global {
}
```

This one only occurred when there was a declare keyword:

```ts
declare namespace Test {
}

// formatted as
declare module Test {
}
```
